pkg/crc/machine/bundle/metadata.go

```diff
@@ -173,6 +174,10 @@ func (bundle *CrcBundleInfo) GetOpenshiftVersion() string {
 	return bundle.ClusterInfo.OpenShiftVersion.String()
 }
 
+func (bundle *CrcBundleInfo) GetPodmanVersion() string {
+	return bundle.Nodes[0].PodmanVersion
```

Comment:
A bit surprising to see `PodmanVersion` depending on the node while `OpenShiftVersion` is part of ClusterInfo?

Reply:
@cfergeau As of now we have single node cluster so podman version can be part of node/cluster but the way our bundle info sample json file is makes it bit confusing when it comes to openshift version because we still have node as list instead having it a single one so in future (which may never happen) when worker and master are part of different node then openshift version still make sense to part of cluster info but podman version is node specific.

Reply:
> when worker and master are part of different node then openshift version still make sense to part of cluster info but podman version is node specific.

Not sure we want to support this without a concrete usecase. If I'm not mistaken, it's also possible to have worker/master nodes with different openshift versions. I'd keep things simple and consistent, and have a single podman version for the whole cluster. When there are multiple nodes, and we need to support different podman versions for each node, then we can revisit this.

Reply:
Still pending..